independent-publisher-connectors/WeatherForecast/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 3
+          },
+          "description": "The city you want to know the weather forecast of",
+          "displayName": "q",
+          "tooltip": "Provide the city along with country code"
+        }
+      },
+      "appid": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": true,
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "The API key of this API",
+          "displayName": "appid",
+          "tooltip": "Provide the API Key"
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",
```

Comment:
@Haimantika Please update it to the default iconBrandColor for all the Independent Publisher Connectors.

`"iconBrandColor": "#da3b01"`